client/webserver/site/src/html/markets.tmpl

```diff
@@ -2,6 +2,7 @@
 <span data-state="sleeping" class="ico-sleeping grey d-hide"></span>
 <span data-state="locked" class="ico-locked grey d-hide"></span>
 <span data-state="unlocked" class="ico-unlocked d-hide"></span>
+<span data-state="syncing" class="ico-sync fs12 d-hide" data-tooltip="placeholder text"></span>
```

Comment:
tooltip still placeholder

Reply:
I replace the text with javascript before it's ever seen. I'm just avoiding an empty string in case I try to evaluate its truthiness somewhere.